### PR TITLE
SPCTR-3637: Adding Filter to add and modify Icon

### PR DESCRIPTION
**Pull request by yash-tingre-10**
### Description
Adding filter to modify and add icons 

### Code Snippet 
`$icons_chunks =  apply_filters( 'uagb_icons_chunks', $icons_chunks );`

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-helper.php [hunk 1]
```diff
@@ -271,8 +271,9 @@ public static function blocks_google_font( $load_google_font, $font_family, $fon
 
 		/**
 		 * Get Json Data.
+		 * Customize and add icons via 'uagb_icons_chunks' filter.
 		 *
-		 * @since 1.8.1
+		 * @since 2.11.3
```
Comment (AbhijitNage123) [inline classes/class-uagb-helper.php:276]: No need to modify since tag for a function.  Add it for filter only

File: README.md [hunk 2]
```diff
@@ -226,6 +226,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Feature: Transform to - Added transform to option from Core Pullquote to Spectra Blockquote and Core List to Spectra Iconlist.
 * Feature: Global - Added a rename option in the list view to customize Spectra block names.
 * Improvement: Core - Updated asset URL generation logic to prevent caching issues.
+* Improvement: Global - Added a filter to modify the icons in the Spectra Icon Picker.
```
Comment (AbhijitNage123) [inline README.md:229]: Changelog entry seems to be added into v2.11.3. it should be in x.x.x

File: classes/class-uagb-helper.php [hunk 3]
```diff
@@ -289,6 +290,8 @@ public static function backend_load_font_awesome_icons() {
 				}
 			}
 
+			$icons_chunks = (array) apply_filters( 'uagb_icons_chunks', $icons_chunks );
```
Comment (adi3890) [inline classes/class-uagb-helper.php:293]: instead of typecasting can we check for is_array

File: classes/class-uagb-helper.php (hunk 1 above)
Comment (yash-tingre-10) [reply classes/class-uagb-helper.php:276]: Okay

Comment (yash-tingre-10) [reply classes/class-uagb-helper.php]: Done with changes

File: classes/class-uagb-helper.php (hunk 3 above)
Comment (yash-tingre-10) [reply classes/class-uagb-helper.php]: Done

File: README.md (hunk 2 above)
Comment (yash-tingre-10) [reply README.md]: Yes that is resolved

**Review by AbhijitNage123 — APPROVED** [review]

**Pull request merged**